modules/logging-input-spec-filter-namespace-container.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-input-spec-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-input-spec-filter-namespace-container_{context}"]
+= Filtering application logs at input by including or excluding namespace or container name
+
+You can include or exclude application logs based on namespace and container name by using the `input` selector.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have created a `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+. Add a configuration for a filter to the `input` spec in the `ClusterLogForwarder` CR.
+
++
+The following example shows how to configure the `ClusterLogForwarder` CR to include or exclude namespaces and container names:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+# ...
+spec:
+  inputs:
+    - name: mylogs
+      application:
+        includes:
+          - namespace: "my-project" # <1>
+            container: "my-container" # <2>
+        excludes: 
+          - container: "other-container*" # <3>
+            namespace: "other-namespace" # <4>
+# ...
+----
+<1> Specifies that logs are only collected from these namespaces.
+<2> Specifies that logs are only collected from these containers.
+<3> Specifies the pattern of namespaces to ignore when collecting logs.
+<4> Specifies the set of containers to ignore when collecting logs.
+
+. Apply the `ClusterLogForwarder` CR by running the following command:
+
++
```

Comment:
Add note or important:  
"The excludes option takes precedence over includes"

Reply:
Done!